### ISSUE TITLE
Generic help  / issue

### ISSUE THREAD

**Issue by pjmccarthyjr** [lucidworks/banana#257]
Sorry this is going to be a bit of a stupid question.  About 6 months ago I setup solr/banana and everything was working great.  I had a dashboard it was working well - it was mostly a POC.  So i didnt touch it for a while and just was tasked with setting up another dashboard.  I fired up solr and went to the existing dashboard and it keeps defaulting back to the /solr/collection1 collection and giving an error.  If i change it to the correct collection it doesnt keep the change. Its showing a mostly a generic dashboard not my custom one.  in my config.js it looks like it was looking for a banana-int index ? I dont see that collection(or core) in my solr dashboard.  
Any thoughts on what to look at to get things back ? As it stands right now I cant customize banana dashboards at all - nothing will save.
Apologies again for the stupid question.

**Comment by nickvasilyev** [comment]
Not a stupid question. 

What I would start with is to check the Dev tools in your browser to see what network requests are going out and check to see if any of them are failing or returning bad data.

**Comment by pjmccarthyjr** [comment]
so that returns errors about not being able to find collection1.  I change the collection to a valid collection and refresh the page and it immediately goes back and looks for collection1.  It looks like its not saving my changes to the dashboard.

**Comment by nickvasilyev** [comment]
You have to change the collection the dashboard is pointing to and save the
dashboard.

**Comment by pjmccarthyjr** [comment]
correct, but its not saving the change - its looking for banana-int(i think)

angular.js:9499 POST http://xxx-xxx-xxx.com:8983/solr/banana-int/update?commit=true 404 (Not Found)(anonymous function) @ angular.js:9499sendReq @ angular.js:9333$http @ angular.js:9124$http.(anonymous function) @ angular.js:9279sjs.client.post @ solr-angular-client.js:65doIndex @ solr.js:8909elasticsearch_save @ dashboard.js:376$scope.elasticsearch_save @ dashLoader.js:98(anonymous function) @ angular.js:6541(anonymous function) @ angular.js:13256Scope.$eval @ angular.js:8218Scope.$apply @ angular.js:8298(anonymous function) @ angular.js:13255p.event.dispatch @ jquery-1.8.0.js:2g.handle.h @ jquery-1.8.0.js:2
angular.js:5930 TypeError: Cannot read property 'status' of undefined
    at dashLoader.js:105
    at wrappedCallback (angular.js:6995)
    at angular.js:7032
    at Object.Scope.$eval (angular.js:8218)
    at Object.Scope.$digest (angular.js:8077)
    at Object.Scope.$apply (angular.js:8304)
    at done (angular.js:9357)
    at completeRequest (angular.js:9520)
    at XMLHttpRequest.xhr.onreadystatechange (angular.js:9490)

**Comment by nickvasilyev** [comment]
That's the issue right there, make sure solr works and it has that
collection.

**Comment by pjmccarthyjr** [comment]
I am pretty sure solr is working, I thinks its just missing that collection, so I need to figure that out.  

Thanks for your help

**Comment by nickvasilyev** [comment]
Yea, make it, there is a schema you need to use on github .
